jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -1020,43 +1091,73 @@ def run(self):
                                 copied_size = os.stat(dark_file).st_size
                                 orig_size = os.stat(filesystem_path(os.path.basename(dark_file))).st_size
                                 if orig_size != copied_size:
```

Comment:
I'm wondering about this. I want to catch cases where there's some major problem with the copying. I've seen cases where the copying stops partway through these large dark files. In that case we need to know. But I've also seen cases where the copied file size is just slightly different than the size of the original file, and in those cases the calculations all seem to work. Maybe some sort of metadata difference or something, which doesn't affect the actual data. But also, darks are generally taken rarely enough that if there's a copying problem, we probably want to do something to use the file in the future. (The next monitor run? Or try again in the current run?)

Reply:
Agreed, I remember when we found this problem, it's a strange one. I think it's something we keep in mind in the future.